scripts/lib.sh

```diff
@@ -41,6 +41,7 @@ get_from_kata_deps() {
 	# through CI. For the kernel, .ci/install_kata_kernel.sh file in tests
 	# repository will pass the kernel version as an override to this function to
 	# allow testing of kernels before they land in tree.
+	CI=${CI:-false}
```

Comment:
This looks ok superficially, but tal at the following:

- https://github.com/kata-containers/tests/blob/master/integration/containerd/cri/integration-tests.sh#L37
- https://github.com/kata-containers/tests/blob/master/integration/containerd/cri/integration-tests.sh#L61
- https://github.com/kata-containers/tests/blob/master/integration/containerd/cri/integration-tests.sh#L71

I think this behaviour needs to be documented and my view is that `CI` should be unset if not running under a CI system. However, I've raised the following so we can discuss and document:

- https://github.com/kata-containers/tests/issues/1649